sova/config/db_loader.py

```
@@ -0,0 +1,185 @@
+    """Sync bridge: load config from DB if possible.
+
+    Returns None (silent fallback) when:
+    - An event loop is already running (dashboard context)
+    - The DB file does not exist on disk
+    - The table does not exist (pre-migration DB)
+    - The DB is unreachable
+    - Any other error
+    """
+    try:
+        asyncio.get_running_loop()
+        return None
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)